src/server/pps/pretty/pretty.go

```diff
@@ -24,7 +24,7 @@ import (
 
 const (
 	// PipelineHeader is the header for pipelines.
-	PipelineHeader = "NAME\tVERSION\tINPUT\tCREATED\tSTATE / LAST JOB\tDESCRIPTION\t\n"
+	PipelineHeader = "PROJECT\tNAME\tVERSION\tINPUT\tCREATED\tSTATE / LAST JOB\tDESCRIPTION\t\n"
```

Comment:
idk how we've done it elsewhere, but should PROJECT be the first column? My naive UX opinion is that the NAME is the most significant piece of information and should therefore come first, even though the PIPELINE fits into a PROJECT

Reply:
Seems like this still needs some design work. Should I keep my current changes in, since it's consistent with listing repos? We can change this once we decide on a proper design.

Reply:
On the one hand, projects contain repos & pipelines, so it makes sense to have them leftmost. OTOH, Alon is right that the individual names are more significant in a list — especially if they are all from the same project. Maybe it would make sense to only display the project column when items from multiple projects are listed?

This probably needs more design work and input from non-engineers. For the time being, I think that it’s probably okay to move forward with this pattern. @maguenza, any objection?